include/mega/logging.h

```diff
@@ -179,15 +187,18 @@ class SimpleLogger
         *mBufferIt = '\0';
         if (continues)
         {
-            *(mBufferIt + 1) = (char)0x1F;
-        }
-        else
-        {
-            *(mBufferIt + 1) = '\0';
+            if (outputted > immediate_threshold)
+            {
+                immediate = true;
```

Comment:
Wouldn't we have to reset `outputted` back to 0 here?

Also: Wouldn't the client be able to determine when to immediate output? Do we really need to pass the `immediate` flag?

Reply:
why the reset? on the contrary, the idea is that from a certain threshold, the message parts are to be outputed directly.

Reply:
The thing is that the client does not know if what's being logged is going to be large or not. that's why I took the decision into the source. The client (i.e. the app, is just consuming some log lines -actually log parts- to have then outputted into the log files). We could consider that the client keeps a track of number of "continuation" messages from the same thread, and move the decision back to it. But it might be overcomplicating that.

Reply:
Oh ok. I might have misunderstood the idea. Maybe add a comment? I would have expected that the chunks that are being immediately output are all the same size but instead the first one only is 1024 and the following ones are 256 (or less).

Reply:
? all chunks are 256 (or less), but the first 4 would be allowed to be kept in the outputting buffer. The fifth will require immediate output before returning the execution. I am increasing the threshold, since this is something we don't want to happen in UI threads, but only in the rare occasions where we have very long lines (requests output).